### PR TITLE
completed html-forms exercise

### PR DESCRIPTION
**Pull request by kvnliang4**
html-forms

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    </div>
+    <select id="option">
+      <option selected="selected">Select an option</option>
+      <option value="Because it's interesting">Because it's interesting</option>
+      <option value="Because it's fun">Because it's fun</option>
+      <option value="Because it's powerful">Because it's powerful</option>
+    </select>
+    <p><label for="favLanguage">Favorite Language (choose one)</label></p>
+
+    <input type="radio" name="language" id="favLanguage" value="HTML">HTML
```
Comment (BrettAlbright) [inline html-forms/index.html]: each of these inputs needs a label, you can wrap the input with the label if desired

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  <title>HTML Forms</title>
+</head>
+
+<body>
+  <form action="???">
+    <h1>Coding Form!</h1>
+    <div>
+      <label for="email">Email</label>
+    </div>
+    <input type="text" id="email">
```
Comment (BrettAlbright) [inline html-forms/index.html]: make sure all of the inputs have a proper `name` attribute

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
looking good! a few small updates are needed

File: html-forms/index.html [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    </div>
+    <select id="option">
+      <option selected="selected">Select an option</option>
+      <option value="Because it's interesting">Because it's interesting</option>
+      <option value="Because it's fun">Because it's fun</option>
+      <option value="Because it's powerful">Because it's powerful</option>
+    </select>
+    <p>Favorite Language (choose one)</p>
+    <label for="favLanguage">
+      <input type="radio" name="language" id="favLanguage" value="HTML">HTML
```
Comment (BrettAlbright) [inline html-forms/index.html:27]: you want to wrap each of these inputs with their own label

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
just a small update for the labels with the radio buttons. overall looks solid!

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**